app/models/host/base.rb

```diff
@@ -351,6 +352,14 @@ def render_template(template:, **params)
       template.render(host: self, **params)
     end
 
+    def to_label
```

Comment:
Where this method is being used?..

Reply:
This is [alias for `to_s` ](https://github.com/theforeman/foreman/blob/6cdb8c7a9ebd790537510213e43537a4a87189d6/app/models/application_record.rb#L51)method.